it/controllers/ConfirmationControllerISpec.scala

```diff
@@ -149,9 +1060,20 @@ class ConfirmationControllerISpec extends IntegrationSpecBase with CreateRequest
           Period("2020, 11, 29".toLocalDate, "2020, 11, 29".toLocalDate))))
       .withUsualHours(List(UsualHours("2020, 12, 26".toLocalDate, Hours(15.86))))
       .withPartTimeHours(List(PartTimeHours("2020, 12, 26".toLocalDate, Hours(1.86))))
-      -> 2381.25,
+      -> 2381.25
   )
 
+  val scenarios: Seq[(UserAnswers, BigDecimal)] = {
+    novemberFourWeeklyScenarios ++
+      novemberMonthlyScenarios ++
+      novemberTwoWeeklyScenarios ++
+      novemberWeeklyScenarios ++
+      novemberVariableFourWeeklyScenarios ++
+      novemberVariableMonthlyScenarios ++
+      novemberVariableTwoWeeklyScenarios ++
+      novemberVariableWeeklyScenarios
+  }
```

Comment:
As discussed, rather than concatenating these Sequences, make this a `Seq[(String, Seq[(UserAnswers, BigDecimal)])]` where the outer string tuple is a name for the scenario.

Then have an outer `foreach` which can output the scenario name. This will help with readabilty of the test output and make it quicker to diagnose any issues with a tests which fails.